test/functional/specs/Logging/C532204.js

```diff
@@ -0,0 +1,45 @@
+import { ClientFunction } from "testcafe";
+import createFixture from "../../helpers/createFixture";
+import configureAlloyInstance from "../../helpers/configureAlloyInstance";
+import {
+  compose,
+  orgMainConfigMain,
+  debugEnabled
+} from "../../helpers/constants/configParts";
+
+const debugEnabledConfig = compose(
+  orgMainConfigMain,
+  debugEnabled
+);
+
+createFixture({
+  title: "C532204: Logged objects can be stringified"
+});
+
+test.meta({
+  ID: "C532204",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+});
+
+const setupLogger = ClientFunction(() => {
```

Comment:
This will cover all scenarios we need from the `triggerAlloyEvent` output?

Reply:
Actually, simply configuring alloy reproduced the bug, but triggering an alloy event creates a lot of other logging statements. This test doesn't exercise all the logging scenarios, but I felt like it was good enough.